src/main/resources/static/css/application.css

```diff
@@ -33,11 +40,9 @@
     background-color: rgba(27, 61, 175, 0.849);
     bottom: 0;
     display: flex;
-
     flex-direction: row;
     list-style: none;
     padding-left: 1rem;
-
```

Comment:
Avoid making changes, especially stylistic ones, in areas outside of where you're working! Please undo these line deletions